components/edit_post/index.ts

```diff
@@ -38,8 +38,8 @@ function mapStateToProps(state: GlobalState) {
     const useChannelMentions = haveIChannelPermission(state, teamId, channelId, Permissions.USE_CHANNEL_MENTIONS);
 
     return {
-        canEditPost: haveIChannelPermission(state, teamId, channelId, deletePermission),
-        canDeletePost: haveIChannelPermission(state, teamId, channelId, editPermission),
```

Comment:
i wonder why it was the other way round previously

Reply:
well, it might have just slipped by the reviews ... its not really of an issue since most users have these permissions anyway, except for guests

Reply:
the values just got mixed up ... easy to oversee that

Reply:
oh yeah